spring-integration-sftp/src/main/java/org/springframework/integration/sftp/session/DefaultSftpSessionFactory.java

```diff
@@ -397,15 +403,34 @@ public void resetSharedSession() {
 		this.sharedSftpClient = null;
 	}
 
+	/**
+	 * This method allows the end-user to provide a custom {@link SftpClient}
```

Comment:
I don't think there is a reason to say on a method that it is a method.
Why just not: `Can be overridden to provide a custom`.
To be honest: I try to pursue "personless" docs as much as possible.
So, something like `you`, `we`, `end-user` are not appropriate words in our docs.

More over the `end-user` in this doc is misleading: the class extension and method override actions are indeed not for users of the compiled programs.